packages/core/src/components/steps-modal/steps-modal.tsx

```diff
@@ -105,7 +122,25 @@ export class AtomStepsModal {
     e.stopImmediatePropagation()
     this.isOpen = false
     this.atomDidDismiss.emit(this.currentStep)
-    this.currentStep = 1
+    this.currentStep = this.customInitialStep ? this.customInitialStep : 1
+  }
+
+  private get secondaryButtonText() {
+    return this.currentStep === this.customInitialStep
+      ? this.cancelButtonText
+      : this.secondaryButtonTextsArray[this.currentStep - 1].trim()
+  }
+
+  private get primaryButtonText() {
+    return this.primaryButtonTextsArray[this.currentStep - 1].trim()
+  }
+
+  private get progress() {
+    if(this.customInitialStep) {
+      return (this.currentStep - this.customInitialStep + 1) / (this.steps + 1 - this.customInitialStep)
```

Comment:
we need some constants to explain this math, man

Reply:
done at[f814758](https://github.com/juntossomosmais/atomium/pull/627/commits/f81475800acd02c38b0999983bb92aff8b013f53)

Reply:
fixed the calc here [d8892bc](https://github.com/juntossomosmais/atomium/pull/627/commits/d8892bc7e0c1a5ebd1b5e3e2c7a05c613fa4c5fd)

Reply:
stepsQuantityA`d`justedToCustomInitial is missing the `d`